src/full.rs

```diff
@@ -0,0 +1,293 @@
+//! Module for functions which ensure full memory safety.
+//!
+//! Functions in this module are guarded from out-of-bounds memory access as
+//! well as from unaligned access, raising errors on both cases. Moreover, only
+//! a [`PodTransmutable`](trait.PodTransmutable.html)) can be used as the
+//! transmute target, thus ensuring full safety.
+//! 
+//! Unless this was previously imposed by certain means, the functions in this
+//! module may arbitrarily fail due to unaligned memory access. It is up to the
+//! user of this crate to 
```

Comment:
to what?

Reply:
Oopsie daisy. This is why I asked for a thorough review. :+1: I'll do something like this:

```suggestion
//! user of this crate to make the receiving data well aligned for the intended target type.
```